setup.json

```diff
@@ -0,0 +1,11 @@
+{
+    "data_path": "Data/franc_swiss_data.csv",
```

Comment:
Ja bym to zostawił jako argument, żeby dało się odpalić od razu eksperymenty dla każdego csv bez potrzeby modyfikacji json'a.